Added clonality, percent on target and percent duplicates (#10)
aidaanva/endorS.py: README.md

# endorS.py
endorS.py calculates various endogenous DNA and deduplication metrics  from samtools flagstat files and print to screen.

## Usage
```bash
usage: python endorS.py [-h] [--version] -r [<samplesfile>.stats] -qF [<samplesfile>.stats] -dedup [<samplesfile>.stats]
```

## Author
Aida Andrades Valtueña (aida.andrades[at]gmail.com)

## Description and options
```bash

> endorS.py calculates Percent on target (aka Endogenous DNA) from samtools flagstat files and print to screen.

Reviewer: ```suggestion
endorS.py calculates percent on target (aka Endogenous DNA) from samtools flagstat files and print to screen.
```

> The Percent on target reported will be different depending on the combination of samtools flagstat provided.

Reviewer: ```suggestion
The percent on target reported will be different depending on the combination of samtools flagstat provided.
```

> This program also calculates clonality (aka Cluster Factor) and percent duplicates when the flagstat file after duplicate removal is provided

Reviewer: ```suggestion
This program also calculates clonality (aka 'cluster factor') and percent duplicates when the flagstat file after duplicate removal is provided
```
